force-app/main/default/classes/RD2_PauseForm_TEST.cls

```diff
@@ -188,6 +188,43 @@ public with sharing class RD2_PauseForm_TEST {
         System.assertNotEquals(null, schedule, 'The Commitment response should be returned');
     }
 
+    @IsTest
+    private static void shouldSavePauseWhenRDIsElevateRecordAndPauseIsRemoved() {
+        RD2_EnablementService_TEST.setRecurringDonations2Enabled();
+        RD2_ElevateIntegrationService_TEST.enableElevateUserPermissions();
+        PS_IntegrationService.setConfiguration(PS_IntegrationServiceConfig_TEST.testConfig);
+
+        npe03__Recurring_Donation__c recurringDonation = rdGateway.getRecords()[0];
+        recurringDonation.CommitmentId__c = 'TEST COMMITMENT';
+        recurringDonation.PaymentMethod__c = RD2_Constants.PAYMENT_PICKLIST_VALUE_CARD;
+        update recurringDonation;
+
+        RD2_PauseForm_CTRL.PauseData pause = new RD2_PauseForm_CTRL.PauseData();
+        pause.rdId = recurringDonation.Id;
+        pause.startDate = null;
+        pause.resumeAfterDate = null;
+        pause.pausedReason = new RD2_PauseForm_CTRL.PausedReason();
+        pause.pausedReason.value = 'Unknown';
+
+        Test.startTest();
+
+        UTIL_Http_TEST.mockNoContentCalloutResponse();
+
+        Exception auraEx;
+
+        try {
+            RD2_PauseForm_CTRL.savePause(JSON.serialize(pause));
+        } catch (Exception ex) {
+            auraEx = ex;
+        }
+
+        Test.stopTest();
+
+        RecurringDonationSchedule__c schedule = [SELECT Id FROM RecurringDonationSchedule__c][0];
+        System.assertEquals(null, auraEx, 'No exception was thrown as a result of the callout');
+        System.assertNotEquals(null, schedule, 'The Commitment response should be returned');
```

Comment:
We aren't actually checking if removing the Pause worked here. Is it possible to do a small integration test where the Schedule itself gets updated? I'd also like to test the new Callout and Response if possible, though I know that happens within `savePause`.